web-api/src/persistence/elasticsearch/docketEntry/getAllDocketEntries.ts

```diff
@@ -0,0 +1,77 @@
+import { MOTION_EVENT_CODES } from '@shared/business/entities/EntityConstants';
+import { QueryDslQueryContainer } from '@opensearch-project/opensearch/api/types';
+import { calculateISODate } from '@shared/business/utilities/DateHandler';
+import { pendingItemCaseSource } from '@web-api/business/useCases/pendingItems/fetchPendingItemsInteractor';
+import { searchAll } from '@web-api/persistence/elasticsearch/searchClient';
+
+export const getAllDocketEntries = async ({
```

Comment:
Can this method either be renamed to indicate that it fetches the pending motions that are older than 180 days or take those filters in as parameters?

Reply:
I think that is a stale file, i did switch it to "getAllPendingMotionDocketEntriesForJudge" file name and method name